.github/workflows/npm-sonar-analysis.yml

```
@@ -61,6 +61,11 @@ jobs:
     - uses: actions/setup-node@v3
       with:
         node-version: ${{ inputs.NODE_VERSION }}
+    - name: Set up JDK 21
+      uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)